templates/zabbix_agentd.conf.erb

```diff
@@ -310,7 +310,7 @@ LoadModulePath=<%= @loadmodulepath %>
 # Mandatory: yes, if TLS certificate or PSK parameters are defined (even for 'unencrypted' connection)
 # Default:
 # TLSAccept=unencrypted
-<% if @tlsaccept %>TLSAccept=<%= @tlsaccept %><% end %>
+<% if @tlsaccept and ( @tlsaccept.instance_of?(Array) or @tlsaccept.instance_of?(String) ) %>TLSAccept=<% if @tlsaccept.instance_of?(Array) %><%= scope.call_function('join', [@tlsaccept, ',']) %><% elsif @tlsaccept.instance_of?(String) %><%= @tlsaccept %><% end %><% end %>
```

Comment:
Something like this (not tested)?

```suggestion
<% if @tlsaccept %>TLSAccept=<%= [@tlsaccept].flatten.join(',') %><% end %>
```

Reply:
@smortex of course it works, thank you very much, it's simple and nice :+1: